client/src/components/MessageList/Style.module.css

```diff
@@ -0,0 +1,20 @@
+.messageContainer {
+  background-color: #f2f2f2;
+  padding: 10px;
+  font-family: inherit;
+  font-size: 18px;
+  width: 60%;
+  height: 50px;
+  margin-top: 25px;
+  margin-bottom: 25px;
+}
```

Comment:
The idea was: 
The gap between message block from A and B is `h`, but,
the gap between message block from A and A  or B and B is `h/2`